lib/nailed.rb

```diff
@@ -17,6 +18,30 @@ def get_colors
     @@colors ||= YAML.load_file(path_to_colors)
   end
 
+  class Cache
+    @@cache ||= Hash.new
+    @@threshold ||= 3600 # 1 hour
```

Comment:
How often do we fetch results from bugzilla?

Reply:
Every hour ;-)